### PR TITLE
boards/msb-430: fix and improve documentation

### PR DESCRIPTION
**Pull request by LudwigKnuepfer**

### CODE REVIEW

File: boards/msb-430/include/board.h [hunk 1]
```diff
@@ -13,7 +13,10 @@
  *
  * <h2>Compontents</h2>
```
Comment (jamesmunns) [inline boards/msb-430/include/board.h]: "Components"

File: boards/msb-430/include/board.h [hunk 2]
```diff
@@ -11,14 +11,20 @@
  * @ingroup     boards
  * @brief       Support for the ScatterWeb MSB-430 board
  *
- * <h2>Compontents</h2>
+ * @details
+ * See http://www.mi.fu-berlin.de/inf/groups/ag-tech/projects/Z_Finished_Projects/ScatterWeb/modules/mod_MSB-430.html for circuit diagram etc.
```
Comment (OlegHahm) [inline boards/msb-430/include/board.h]: Yes, the URL is long, but maybe we can still have at least one line break after it.

Comment (LudwigKnuepfer) [inline boards/msb-430/include/board.h]: off course

**Comment by OlegHahm** [comment]
ACK

**Comment by kaspar030** [comment]
false pos from riot ci. argl. :)

**Comment by kaspar030** [comment]
(the build errors are unrelated, still it should fail on them)

**Comment by LudwigKnuepfer** [comment]
https://github.com/RIOT-OS/RIOT/pull/3765 << this was merged without CI and seems to offend if your CI does not lie

**Comment by LudwigKnuepfer** [comment]
.. or maybe not, I should go to bed ;)

**Comment by kaspar030** [comment]
&go.

**Pull request merged**